core/test/oracle/ResultComputation.js

```diff
@@ -15,17 +15,17 @@ contract("ResultComputation", function(accounts) {
     await resultComputation.wrapAddVote(priceOne, web3.utils.toWei("5"));
     // Frequency table: priceOne->5. Cutoff: 2.5.
     let resolved = await resultComputation.wrapGetResolvedPrice(minVoteThreshold);
-    assert.isTrue(resolved.isResolved);
-    assert.equal(resolved.price, priceOne);
+    assert.isTrue(resolved[0]);
```

Comment:
@mrice32, @nicholaspai and @ptare if we make this change we can't directly refer to variables if the function called returns more than one variable. I'd like to get your input before making this change. Options are:
1) revert to returning named variables. This would mean that we are inconsistent. might be acceptable iff the function returns two separate variables. rule is then that if it has two we use named variables.
2) accept the small hit in code readability and ensure to always add comments for code readability.

On another note I know these semantic PRs have spent a lot of time because a fair amount of it's style related but I think it's good because we establish an agreed apon pattern that we use going forward for consistency sake.

Reply:
After talking to @mrice32 and @nicholaspai about this we converged on two different opinions between readability and consistency. @mrice32's preference is readability and @nicholaspai's preference is consistency.

While it's inconsistent to have named return types in some functions and not others, I feel the benefit in integrating with the contracts with simple named values outways the inconsistency cost. In all other cases having a named return variable is the desired pattern. 

Going forward as a styling rule we will now always used named return variables in all cases EXCEPT when returning multiple variables from a Solidity function.

Reply:
@chrismaree That makes sense